### ISSUE TITLE
for homework3, is there any difference with https://github.com/xueyuelei ???

### ISSUE THREAD

**Issue by FlyHighInSky** [yangxj59/Pattern-Recognition#2]

**Comment by yangxj59** [comment]
Sorry，Mr.Hu.I can not configure theano environment.So i use the homework of xueyuelei.But I made some notes different from his.

**Comment by FlyHighInSky** [comment]
Yes, I noted the diffenent comments, but you can tell me the issues that you encountered. I suggest you to pass me the screenshot of the issues.
